tests/drivers/memc/ram/testcase.yaml

```diff
@@ -3,3 +3,7 @@ tests:
     tags: drivers memc
     depends_on: memc
     filter: dt_compat_enabled("st,stm32-fmc-sdram")
+  drivers.memc.smc_sram:
+    tags: drivers memc
+    depends_on: memc
+    filter: dt_compat_enabled("atmel,sam-smc")
```

Comment:
twister is skipping `sam4s_xplained` board. It should be possible build locally with, I think:
`./scripts/twister -p sam4s_xplained -t tests/drivers/memc/ram`

Reply:
Shouldn't this work? I'm not sure how to fix/improve this.
I thought about dropping the `dt_compat_enabled` filter entirely and check the nodelabels instead:

```yaml
tests:
  drivers.memc.ram:
    tags: drivers memc
    depends_on: memc
    filter: dt_nodelabel_enabled("sdram1") or dt_nodelabel_enabled("sdram2") or
            dt_nodelabel_enabled("sram1") or dt_nodelabel_enabled("sram2")

```

Any suggestions?

Reply:
No clue about it. Please, take a look at Discord -> Infrastructure channel.

Reply:
@nandojve I've added `sam4s_xplained` as an integration platform, CI should build it now.